source/guides/basics/mechanisms.md

```diff
@@ -1,49 +1,150 @@
 
 ### The Adjust Command
 
-The `adjust` command is primarily used to adjust entities that are already spawned or otherwise have a unique identifier.
+Mechanisms can be used in more than just script containers! If you need to alter the value of a mechanism, the `adjust` command is the way to go. The `adjust` command is primarily used to adjust entities that are already spawned or otherwise have a unique identifier.
+
+For example, if we want to write a script that increases the max health of the swordsman we made above, we could do it like this:
+
+```dscript_green
+undead_golden_swordsman_growth:
+    type: world
+    events:
+        on entity killed by undead_golden_swordsman:
+        - define current_damage <context.damager.attribute_value[generic_attack_damage]>
+        - adjust <context.damager> attributes:generic_attack_damage/<[current_damage].add[1]>
+```
+
+This script will give any `undead_golden_swordsman` that kills another entity an additional 1 attack damage for each kill it achieves.
 
 #### The AdjustBlock command
 
-The `adjustblock` command is a variant of normal `adjust` specifically to adjust `material` mechanisms on blocks.
+The `adjustblock` command is a variant of the normal `adjust` command. It's built specifically to adjust `material` mechanisms on blocks <span class="parens">([a list of which can be found here](https://one.denizenscript.com/denizen/mecs/materialtag))</span>.
+
+While the applicability of `adjustblock` is more niche, it can still be quite useful. For a quick usage example, you can use this script to force the plant block you are looking at to immediately change to its maximum age <span class="parens">(i.e. to cause newly-planted wheat to fully grow)</span>:
+
+```dscript_green
+grow_plant:
+    type: task
+    script:
+    - if <player.cursor_on.material.is_ageable>:
+        - adjustblock <player.cursor_on> age:<player.cursor_on.material.maximum_age>
+```
+
+As before, you can use this script by typing `/ex run grow_plant` in-game.
 
 #### How To Adjust Items
 
-**TODO: Explain adjust/adjustblock more, including how `inventory adjust` works, and including what a 'unique identifier' is**
+You can also adjust the properties of items in inventories. Doing so uses the `inventory` command with the `adjust` argument.
+
+You might think that the correct way to modify an item held by a player is by adjusting the item itself:
+
+```dscript_red
+magical_flint_bad:
+    type: world
+    events:
+        on player right clicks block with:flint:
+        - adjust <player.item_in_hand> material:diamond
+```
+
+However, this will only adjust the *generic concept* of the item in the player's hand. If you're not adjusting a specific item that exists in the world somehow, nothing is going to change in the world! Instead, you need to use the `adjust` argument of the `inventory` command to adjust the specific item that you want to change.
```

Comment:
I think this could use some further explanation, about how the tag returns just `flint`, which means... just any flint, and doesn't refer to that one specific piece of flint in the player's inventory.